src/ui/overlay.html

```diff
@@ -88,6 +96,9 @@
     <img src="../../assets/hand.svg" alt="">
     <span></span>
   </div>
+  <div id="status">
+    <span id="statusIndicator"></span>
+  </div>
```

Comment:
If the message if absolutely centered, we should not need 2 nodes here

Reply:
Absolutely true, removing statusIndicator span